src/main/java/org/galatea/starter/service/IexClient.java

```diff
@@ -20,7 +25,7 @@ public interface IexClient {
    *
    * @return a list of all of the stock symbols supported by IEX.
    */
-  @GetMapping("/ref-data/symbols")
+  @GetMapping("/ref-data/symbols?token=pk_4e957a4821324131ae5619df8c683985") //hardcoded private token to the mappings
```

Comment:
It's good you are using the API token, but can you find a way to configure and access this as an environment variable rather than including it directly in your code?